proto/ligato/vpp/nat/nat.proto

```diff
@@ -75,7 +75,26 @@ message DNat44 {
     repeated IdentityMapping id_mappings = 3;
 }
 
-message VirtualReassembly {                 /* NAT virtual reassembly */
+/* Local network interfaces enabled for NAT44. */
+message Nat44Interface {
+    string name = 1;                        /* (logical) Interface name. */
```

Comment:
Comments in proto should be line comments located above the particular field/message. Otherwise they don't end up in generated file.

Reply:
Good idea. I was keeping the style of comments in the existing proto. I can refactor it completely.